src/nbl/asset/utils/CIESProfileParser.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018-2024 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef __NBL_ASSET_C_IES_PROFILE_PARSER_H_INCLUDED__
+#define __NBL_ASSET_C_IES_PROFILE_PARSER_H_INCLUDED__
+
+#include "nbl/asset/utils/CIESProfile.h"
+
+namespace nbl
+{
+    namespace asset
+    {
+        class CIESProfileParser 
+        {
+            public:
+                CIESProfileParser(char* buf, size_t size) { ss << std::string(buf, size); }
+
+                const char* getErrorMsg() const { return errorMsg; }
+                bool parse(CIESProfile& result);
```

Comment:
document the IES spherical coordinates to illuminant target, and whether the X-Y (cos(phi), sin(phi)) are LH or RH

Reply:
resolved